operations/mimir-mixin-tools/serve/run.sh

```diff
@@ -4,7 +4,9 @@
 set -e
 
 SCRIPT_DIR=$(cd `dirname $0` && pwd)
-DOCKER_CONTAINER_NAME="mixin-serve-grafana"
+# Ensure we run recent Grafana.
+GRAFANA_VERSION=11.0.0
```

Comment:
Note that, as of today, `latest` tag is Grafana 10.x, which isn't what we want in dev